repo/packages/C/ceph/0/config.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
I think we should also explain the steps in prereqs in dcos/example along with screenshots.